src/ServiceControl.Audit.Persistence/IPersistence.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace ServiceControl.Audit.Persistence
+{
+    using Microsoft.Extensions.DependencyInjection;
+
+    public interface IPersistence
+    {
+        IPersistenceLifecycle CreateLifecycle(IServiceCollection serviceCollection);
```

Comment:
`CreateLifecycle` sounds a bit off as a name because one doesn't understand why it gets an `IServiceCollection` as an argument. `Configure` seems to better align with MS APIs accepting a service collection.

Reply:
agreed, will come up with a better name